### PR TITLE
Comments addressed

### PR DESCRIPTION
**Pull request by mystiberry-okta**
@yuliu-okta and @cwu-okta this is the latest PR for oauth2 and oidc, addressing Yu's great comments.

I have to publish starting around 2:30 PM, but please continue to comment even if this is merged.

### CODE REVIEW

File: _source/_docs/api/resources/oauth2.md [hunk 1]
```diff
@@ -368,7 +368,7 @@ Irrespective of the response type, the contents of the response is always one of
 Parameter         | Description                                                                                        | DataType  | 
 ----------------- | -------------------------------------------------------------------------------------------------- | ----------| 
 id_token          | The ID Token JWT contains the details of the authentication event and the claims corresponding to the requested scopes. This is returned if the *response_type* includes *id_token*.| String    | 
-access_token      | The *access_token* that is used to access the [`/oauth2/v1/userinfo`](/docs/api/resources/oidc.html#get-user-information) endpoint. This is returned if the *response_type* included a token. Unlike the ID Token JWT, the *access_token* structure is specific to Okta, and is subject to change.| String  |
+access_token      | The *access_token* that is used to access the resource. This is returned if the *response_type* included a token. Unlike the ID Token JWT, the *access_token* structure is specific to Okta, and is subject to change.| String  |
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oauth2.md]: remove "Unlike the ID Token JWT, the *access_token* structure is specific to Okta, and is subject to change."

File: _source/_docs/api/resources/oidc.md [hunk 2]
```diff
@@ -319,6 +319,182 @@ Please note the following:
...
+
+> Note; [ID Tokens](oidc.html#id-token) are also valid, however, they are usually validated on the service provider or app side of a flow.
+
+#### Request Parameters
+
+The following parameters can be posted as a part of the URL-encoded form values to the API.
+
+Parameter       | Description                                                                                         | Type       |
+----------------+-----------------------------------------------------------------------------------------------------+------------|
+token           | An access token or refresh token.                                                                   | String     |
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oidc.md]: An access token, ID token or refresh token

File: _source/_docs/api/resources/oidc.md [hunk 3]
```diff
@@ -387,7 +563,7 @@ Standard open-source libraries are available for every major language to perform
 
 #### Alternative Validation 
 
-You can use an [OAuth 2.0 introspection request](/docs/api/resources/oauth2.html#introspection-request) for validation if you have the API Access Management feature.
+You can use an [introspection request](#introspection-request) for validation if you have the API Access Management feature.
```
Comment (yuliu-okta) [inline _source/_docs/api/resources/oidc.md]: Remove "if you have the API Access Management feature"

Comment (mystiberry-okta) [inline _source/_docs/api/resources/oidc.md]: Missed that, sorry!

**Pull request merged**